release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1267,6 +1267,26 @@ sourceStrategy:
 [id="ocp-4-12-networking-bug-fixes"]
 ==== Networking
 
+* Previously, routers left in the terminating state delayed the `oc cp` command, which delayed the must gather until the terminating pod was terminated. With this update, a timeout is set for each `oc cp` command resulting in the must gathers not being delayed by terminating pods. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2103283[*BZ#2103283*])
```

Comment:
```suggestion
* Previously, routers that were in the terminating state delayed the `oc cp` command which would delay the `oc adm must-gather` command until the pod was terminated. With this update, a timeout for each issued `oc cp` command is set to prevent delaying the `must-gather` command from running. As a result, terminating pods no longer delay `must-gather` commands. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2103283[*BZ#2103283*])
```